categorization-state-machine/README.md

```diff
@@ -0,0 +1,82 @@
+# Message categorizing with Amazon Bedrock and Amazon SQS
+
+This workflow demonstrates how you can use Amazon Bedrock to categorize messages and then send them to an Simple Queue Service queue for further processing. The AWS Serverless Application Model (SAM) template deploys the AWS Step Functions state machine which takes care of the steps related to the workflow. 
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the AWS Pricing page for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- Create an AWS account if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- AWS CLI installed and configured
```

Comment:
- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured